src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/EventCounterPipeline.cs

```diff
@@ -15,6 +15,7 @@ internal class EventCounterPipeline : EventSourcePipeline<EventPipeCounterPipeli
     {
```

Comment:
If we are now using this Pipeline to also process events from the MetricsEventSource then EventCounterPipeline sounds a little misleading now. System.Diagnostics.Metrics != EventCounters. Perhaps we want to call this the MetricsPipeline or the CountersPipeline to try signaling that is more generic and handling multiple different APIs?